mock/src/main/java/com/tngtech/keycloakmock/impl/handler/TokenRoute.java

```diff
@@ -114,4 +144,13 @@ private String toTokenResponse(String token, String sessionId) {
         .put(SESSION_STATE, sessionId)
         .encode();
   }
+
+  private String toTokenResponse(String token) {
+    // With client credentials flow, there is no refresh token and no session
```

Comment:
I know that this is what the specification says, but unless there are actual client side issues, I'd like to keep the code as simple as possible, and just return the same response everywhere.